.github/workflows/upgrade-operator.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: "Check current state"
         run: |
           git config user.name "GitHub Actions Bot"
-          git config user.email "<>"
+          git config user.email "noreply@github.com"
```

Comment:
Is this an actual noreply email address provided by Github, or it is just a placeholder?

Reply:
It's the one that people seem to use.  I've discovered that we aren't going to be able to get the CLA to pass with this robot account anyway, so there's still more to do here